suite/tests/client-interface/drx-scattergather.c

```diff
@@ -154,6 +154,22 @@ test_avx2_vgatherqpd(uint32_t *ref_sparse_test_buf, uint32_t *test_idx32_vec,
 #    define SPARSE_TEST_BUF_SIZE_U32 (SPARSE_FACTOR * ZMM_REG_SIZE / sizeof(uint32_t))
 #    define POISON 0xf
 
+static bool
+test_avx512_mask()
+{
+#    ifdef UNIX
+    /* XXX i#2985: add check to non-UNIX systems. */
+    uint32_t k_buf[2];
+    memset(k_buf, 0, sizeof(k_buf));
+    uint32_t ref_buf[2];
+    memset(ref_buf, 0, sizeof(ref_buf));
+    __asm__ __volatile__("kmovw %%k1, %0" : : "m"(k_buf));
+    if (memcmp(k_buf, ref_buf, 2) != 0)
```

Comment:
So this is checking that the mask is all 0's.  Maybe the function name and ref_buf should reflect that.  I kept looking for where it gets the real value for ref_buf.

Reply:
Done.